src/main/markdown/articles/gwt-iphone.md

```diff
@@ -180,7 +180,7 @@ two HTTP round-trips.
 The Feed Reader needs to get its feed data from somewhere. Enough
 of the
 [Google
-AJAX Feed API](//developers.google.com/feed/) was imported with Java bindings using the
+AJAX Feed API](http://developers.google.com/feed/) was imported with Java bindings using the
```

Comment:
use https directly, http://developers.google.com/ redirects to https://developers.google.com/